docs/learning/workshop_a_precision_converter_fpga_integration_journey/index.rst

```diff
@@ -0,0 +1,693 @@
+A Precision Converter FPGA Integration Journey
+==============================================
+
+Agenda
+~~~~~~
+
+1. Introduction
+2. SPI Engine Architecture
+3. What Is the Use Case?
+4. System Build
+5. System Evaluation
+6. Conclusions
+
+Introduction
+~~~~~~~~~~~~
+
+**Customer Journey**
+
+.. figure:: intro_customer_journey_1.png
+   :align: center
+
+Tools / Platforms are a customer choice, ADI ports to “current” development
+environment/kit.
+
+.. figure:: intro_customer_journey_2.png
+   :align: center
+
+Maintenance is an ADI burden. Customer don’t start their designs at the same
+time, and want to use the latest and greatest.
+
+**COS Reference Design “Donut Hole” Strategy**
```

Comment:
```suggestion
Maintenance is an ADI burden. Customer don't start their designs at the same
time, and want to use the latest and greatest.

**COS Reference Design "Donut Hole" Strategy**
```

- [ ] That type of single and double quotes messes with the pdf generation if I remember correctly, so please change them to these, in the whole page:

' 
and 
"

- [ ] Also, maybe it would be better if these would be made as subsections

Reply:
Fixed quotes